app/utils/middlewares.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-apollo-client/utils/middlewares';
```

Comment:
I don't think we should add this file in `app/`. The user has to import it wherever they use it anyway, so we may as well have them import it as `'ember-apollo-client/utils/middlewares'` to avoid polluting up their `utils/` namespace.

Thoughts?